### DISCUSSION TITLE
Group creation and administration by non-admins

### DISCUSSION THREAD

**Discussion by iexos** [outline/outline#5825] [Ideas]
I would love if normal users would have more options to self-organize their groups within a workspace. Ideally, Users could create groups on their own and add any members they want. Users could leave groups on their own as well. This would probably require a "group admin" membership level of a group, though for me personally it would be fine if all members could modify the group.

**Comment by tommoor** [comment]
This seems a-typical, if all your users are acting like admins can you just make them admins?

**Comment by iexos** [comment]
I believe everything that can be done with groups could in principle done without groups, e.g. creating collections and giving access to everyone I would like to have it. Groups are a convenience feature to make that more simple. Workspace admins have much more rights than group administration, that I wouldn't want everyone to have.

**Reply by iexos** [reply]
To clarify, users could only modify groups they themselves are members of (and may have the status of group admin). This can reduce friction in organizations that try to mitigate hierarchies and use self-organization like sociocracy or similar. The workspace admin would still be able to ensure a clear structure.

**Reply by tommoor** [reply]
Not against a group admin role 👍

**Reply by krupper** [reply]
We need this feature as well.
